### ISSUE TITLE
Telegram notifications not working

### ISSUE THREAD

**Issue by TheSpad** [crazy-max/diun#162]
### Steps to reproduce this issue

1. Configure diun to send telegram notifications
2. Wait for update check to find updated image

### Expected behaviour

Telegram notification should be sent on new image update available. 

### Actual behaviour

Telegram notification is not send and error is logged. Last time I can be sure this was working was with version 4.3.0; last successfullly sent notification was August 8th. Running `diun --test-notif` with the same config .yml is successful. Config/Compose has not been changed since last successful notification was sent.

## Configuration

* Diun version : 4.4.1
* Platform (windows/linux) : Linux
* System info (type `uname -a`) :  Linux docker1 5.4.0-40-generic #44-Ubuntu SMP Tue Jun 23 00:01:04 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

Compose
```yaml
version: "3.2"

services:
  diun:
    image: crazymax/diun:latest
    container_name: diun
    hostname: diun_docker1
    volumes:
      - diun_data:/data
      - ./diun.yml:/diun.yml:ro
      - /var/run/docker.sock:/var/run/docker.sock:ro
    networks:
      - public
    environment:
      - "TZ=Europe/London"
      - "LOG_LEVEL=debug"
      - "LOG_JSON=false"
      - "CONFIG=/diun.yml"
    restart: unless-stopped
    labels:
     - diun.enable=true

volumes:
    diun_data:
        external: true

networks:
    public:
        external: true
```
Diun config
```yaml
db:
  path: diun.db

watch:
  workers: 10
  schedule: "0 */6 * * *"
  firstCheckNotif: true

notif:
  telegram:
    token: 1000000000:AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
    chatIDs:
      - -1000000000000

providers:
  docker:
    watchStopped: true
```
### Logs

```
Wed, 26 Aug 2020 15:36:14 BST INF Image update found image=docker.io/linuxserver/unifi-controller:latest provider=docker
Wed, 26 Aug 2020 15:36:14 BST DBG Manifest saved to database image=docker.io/linuxserver/unifi-controller:latest provider=docker
Wed, 26 Aug 2020 15:36:14 BST DBG Sending telegram notification... image=docker.io/linuxserver/unifi-controller:latest
Wed, 26 Aug 2020 15:36:14 BST ERR Telegram notification failed error="Bad Request: can't parse entities: Can't find end of the entity starting at byte offset 186" image=docker.io/linuxserver/unifi-controller:latest
```

**Comment by crazy-max** [comment]
@TheSpad Looks like an issue [parsing the markdown message](https://github.com/crazy-max/diun/blob/master/internal/notif/telegram/telegram.go#L48). Can you give me the output of `hostname`?

**Comment by TheSpad** [comment]
The hostname in this instance is "diun_docker1", though I have two other hosts (also with fixed hostnames) that are showing the same issue.

**Comment by crazy-max** [comment]
@TheSpad Ok `_` is the culprit here. I will make some changes to escape "incomplete" markdown syntax. Telegram servers are very strict about this.

**Comment by crazy-max** [comment]
@TheSpad Should be fixed (v4.5.0)

**Comment by TheSpad** [comment]
Have pulled the new version and will do some tests. Cheers.
